com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -288,31 +326,22 @@ private void OnSceneLoaded(Guid switchSceneGuid, Stream objectStream)
 
             if (m_NetworkManager.IsServer)
             {
-                OnSceneUnloadServer(switchSceneGuid);
+                OnServerLoadedScene(switchSceneGuid);
             }
             else
             {
-                OnSceneUnloadClient(switchSceneGuid, objectStream);
+                OnClientLoadedScene(switchSceneGuid, objectStream);
             }
         }
 
-        private void OnSceneUnloadServer(Guid switchSceneGuid)
+        private void OnServerLoadedScene(Guid switchSceneGuid)
```

Comment:
while you're doing some small refactors here, do you mind renaming all `switchSceneGuid` and other `switchXXX` with non-`switch` versions such as `sceneGuid` instead of `switchSceneGuid`? I believe it wouldn't harm the understanding but fix syntax-highlighter getting confused :)

Reply:
Good idea...

Reply:
Will see if there is a better term for this... I ignored some of this stuff because the later phases of the proposal would effectively remove all of that.